### ISSUE TITLE
Use TIDs instead of data submitter IDs (both new and existing samples)

### ISSUE THREAD

**Issue by e-t-k** [UCSC-MedBook/MedBook-PatientCare#89]
all samples from prospective cases are going to be assigned TIDs (or have been already? not sure.)

Need to
- convert all existing samples to the TID (or otherwise assign them a TID)
- use TIDs henceforth when importing new samples

By user request.

**What do users want to be able to do?**
for example, they might:
- download TSV that populates with TIDs instead of submitter IDs
- when running OA, select sample by TID
- create sample group filters using a form that references samples by TID

Essentially ... we want to be able to make data sets with the desired sample TIDs but instead of wrangling in the expression data, just use the expression data of another data set.

**Suggested workflow:**
- User goes to wrangler and uploads a "sample alias" file, selecting desired Data Set.
- also enter the description of what the alias is, eg "TID"
  File is 2 columns: sample ID : sample alias.
- Aliases will go under the same study label as the original IDs
- When downloading, you select which labels to use
- when running OA -- maybe show all of both labels and aliases to pick from?

**Comment by mokolodi1** [comment]
Would we not just want to rename the samples? We could have a bulk rename feature. 

Also, what is a TID?

**Comment by rcurrie** [comment]
TID = Treehouse Identifier, de-identified durable identifier for all inputs data files managed by Treehouse.

I don’t think we should have a rename feature as TID’s must be durable and immutable.

We may want to do a one time re-import when there are the new TID’s, will bring up at today’s meeting.

**Comment by e-t-k** [comment]
> We may want to do a one time re-import when there are the new TIDs

Agreed. If samples can have two "equally canonical" labels , this wouldn't be necessary; but if it's implemented such that samples have, eg, one canonical "label" and one (or more) "aliases", then the canonical label should be TID instead of submitter ID which requires re-import of current samples.
